package.json

```diff
@@ -45,6 +45,7 @@
   },
   "dependencies": {
     "@flowcrypt/fine-uploader": "5.16.4",
+    "@types/thunderbird-webext-browser": "^109.0.4",
```

Comment:
seems like these types are not used anywhere, dependency can be removed

Reply:
Ah yes - I pre-added this so that wouldn't miss them later one. This is pretty much useful when I started calling the `browser` object known by the Thunderbird API.

I'll check and remove it when unused. thanks!